sdk/keyvault/azure-security-keyvault-certificates/CHANGELOG.md

```diff
@@ -1,4 +1,13 @@
 # Release History
+## 4.0.0-beta.7 (2019-12-19)
+- `beginDeleteCertificate` and `beginRecoverDeletedCertificate` polling operation to be `SUCCESSFULLY_COMPLETED` on 403.
```

Comment:
`beginDeleteCertificate` and `beginRecoverDeletedCertificate` methods now return a poll response with a status of `SUCCESSFULLY_COMPLETED` when service returns 403 status.